container-conf/build.sh

```diff
@@ -8,31 +8,28 @@ build_is_public=1
 # POSIT container-beamsim-jupyter
 # nvidia-container-runtime needs the environment vars "OCI"
 build_dockerfile_aux='ENV NVIDIA_DRIVER_CAPABILITIES compute,utility
-ENV NVIDIA_REQUIRE_CUDA "cuda>=11.1 brand=tesla,driver>=410"'
+ENV NVIDIA_REQUIRE_CUDA "cuda>=12.2 brand=tesla,driver>=525.60.13"'
```

Comment:
I wonder about these lines here. Can they be more flexible, e.g. ">=525"

Reply:
cuda>=12.2 is for tensorflow. 

driver>=525.60.13  is for cuda toolkit 12.x https://docs.nvidia.com/deploy/cuda-compatibility/